x-pack/plugins/cloud_security_posture/public/pages/benchmarks/use_csp_benchmark_integrations.ts

```diff
@@ -40,7 +40,7 @@ export const useCspBenchmarkIntegrations = ({
 
   return useQuery(
     [QUERY_KEY, query],
-    () => http.get<ListResult<Benchmark>>(BENCHMARKS_ROUTE_PATH, { query }),
+    () => http.get<ListResult<Benchmark>>(BENCHMARKS_ROUTE_PATH, { query, version: '1' }),
```

Comment:
i should have commented about that on the previous pr as well but we should probably have a shared constant for the version number, so we can manage the version in one place.